src/components/layout/layout.module.css

```diff
@@ -0,0 +1,10 @@
+.container {
+    color: #FFFEFE;
+    width: 100%;
+}
+
+.content {
+    margin: 0 auto;
+    max-width: 1200px;
```

Comment:
since the codebase is pretty small now this is minor but you could consider using [bootstrap containers](https://getbootstrap.com/docs/4.4/examples/grid/#containers) in a followup PR to get this responsive feel.

The cons to Bootstrap are that its a decent sized bundle and you probably won't use all of it. But the intent of Bootstrap is just to help you get through the "bootstrap" phase of building your app and then down the road they expect you to prune/minify the Bootstrap bundle to only what you need.

The pros of Bootstrap is that it's a styling framework that lets you share common styles (responsive widths, theme colors, etc.) throughout the app. It also helps reduce the amount of repeated css.

There's different schools of thought on this though so if you prefer to continue writing separate css modules for each component that works too!

Reply:
Sounds good, I will keep Bootstrap in mind for a later PR.